### DISCUSSION TITLE
How to write a volume plugin?

### DISCUSSION THREAD

**Discussion by geniys** [containers/podman#23762] [Q&A]
I want to make a custom volume plugin but I can't find any resources about it.
If anyone has this information, I would appreciate your help.

**Comment by rhatdan** [comment]
I believe you should be able to follow the docker volume plugin definition.  We don't see much demand for these, so not much documentation.

What do you plan to do in this plugin?

**Reply by geniys** [reply]
I am looking for a way to limit the volume size without using XFS. Thank you. I will check the Docker plugin documentation.
